webSocket/data/operations.go

```diff
@@ -45,23 +45,23 @@ func (ot OperationType) String() string {
 }
 
 // Uint32 will return the uint32 representation of the operation
-func (ot OperationType) Uint32() uint32 {
+func (ot PayloadType) Uint32() uint32 {
 	return uint32(ot)
 }
 
 const (
-	// OperationSaveBlock is the operation that triggers a block saving
-	OperationSaveBlock OperationType = 0
-	// OperationRevertIndexedBlock is the operation that triggers a reverting of an indexed block
-	OperationRevertIndexedBlock OperationType = 1
-	// OperationSaveRoundsInfo is the operation that triggers the saving of rounds info
-	OperationSaveRoundsInfo OperationType = 2
-	// OperationSaveValidatorsPubKeys is the operation that triggers the saving of validators' public keys
-	OperationSaveValidatorsPubKeys OperationType = 3
-	// OperationSaveValidatorsRating is the operation that triggers the saving of the validators' rating
-	OperationSaveValidatorsRating OperationType = 4
-	// OperationSaveAccounts is the operation that triggers the saving of accounts
-	OperationSaveAccounts OperationType = 5
-	// OperationFinalizedBlock is the operation that triggers the handling of a finalized block
-	OperationFinalizedBlock OperationType = 6
+	// PayloadSaveBlock is the payload type that triggers a block saving
```

Comment:
these have nothing to do with the implementation of the web socket host. Suggestion to keep only the 
```GO
// PayloadType defines the type to be used to group web socket operations
type PayloadType uint8
``` 
definition. The constant definitions should be done elsewhere.

Reason: these are application-specific values. A convention between our node and the indexer, not related to the websocket hosts implementations

Reply:
refactored